packages/core/src/components/cv-checkbox/cv-checkbox.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <div
-    style="{position: 'relative'}"
     class="cv-checkbox"
-    :class="[`${carbonPrefix}--checkbox-wrapper`, { [`${carbonPrefix}--form-item`]: formItem }]"
+    :class="[
+      `${carbonPrefix}--checkbox-${inline ? 'inline' : 'wrapper'}`,
```

Comment:
should be `--checkbox--` instead of `--checkbox-`

Reply:
the wrapper class is `--checkbox-wrapper` while the inline one is `--checkbox--inline` (with double dashes)

Reply:
nice